xtask/src/commands/package/mod.rs

```diff
@@ -40,8 +40,7 @@ impl Package {
             self.output.to_owned()
         } else if self.output.is_dir() {
             self.output.join(format!(
-                "{}-{}-{}-{}.tar.gz",
-                RELEASE_BIN,
+                "router-{}-{}-{}.tar.gz",
```

Comment:
not sure what is happenning so i ll ask away: is line 25 still correct? 

```
let release_path = TARGET_DIR.join("release").join(RELEASE_BIN);
```

Reply:
It's good